.github/workflows/maven.yml

```diff
@@ -15,16 +15,12 @@ jobs:
 
     runs-on: ubuntu-latest
 
-    strategy:
-      matrix:
-        java-version: ['1.8', '11']
```

Comment:
I prefer sticking to the build matrix, but 11 and 17 are nowadays probably more meaningful than 8 and 11.